default/python/auth/auth.py

```diff
@@ -72,5 +72,6 @@ def list_players(self):
                 psd.player_name = player_name
                 psd.empire_name = player_name
                 psd.starting_species = "RANDOM"
+                psd.starting_team = -1
```

Comment:
Probably better to initialize with `INVALID_PLAYER_ID` to avoid "magic number" in code.

Reply:
For python `INVALID_ID = -1` defined in the `AIDependencies.py`. Also it isn't player id at all. Maybe add new constant?

Reply:
Another constant also OK.

Reply:
I've added `NO_TEAM_ID` constant.